PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/SparkUserException.kt

```diff
@@ -46,6 +47,10 @@ object SparkUserExceptionFactory : ClassifiedExceptionFactory() {
         } else if (exp is ADLException && exp.httpResponseCode == 403) {
             val hintMsg = "\nPlease make sure user has RWX permissions for the storage account"
             SparkUserException(ADLException("${exp.remoteExceptionMessage}$hintMsg"))
-        } else null
+        } else if(exp is AuthException && exp.error.equals("invalid_grant")){
+            val hintMsg = "\nPlease make sure Azure Toolkits for IntelliJ IDEA and Eclipse client App Id is correctly added to the cluster related client App Id"
+            SparkUserException(AuthException("${exp.errorMessage}$hintMsg"))
```

Comment:
Add hint link in next iteration

Reply:
What does the `added to the cluster related client App Id` mean?